TOC-tidb-cloud.md

```diff
@@ -137,7 +137,7 @@
   - [Changefeed Overview](/tidb-cloud/changefeed-overview.md)
   - [To MySQL Sink](/tidb-cloud/changefeed-sink-to-mysql.md)
   - [To Kafka Sink](/tidb-cloud/changefeed-sink-to-apache-kafka.md)
-  - [To TiDB Cloud Sink](/tidb-cloud/changefeed-sink-to-tidb-cloud.md)
+  - [To TiDB Cloud Sink (beta)](/tidb-cloud/changefeed-sink-to-tidb-cloud.md)
```

Comment:
```suggestion
  - [To TiDB Cloud Sink (Beta)](/tidb-cloud/changefeed-sink-to-tidb-cloud.md)
```

Please also update L2 and L6 to "Sink to TiDB Cloud (Beta)"

https://github.com/pingcap/docs/blob/d01a93dd22e64a74f305f62524dbab5f6fda70c3/tidb-cloud/changefeed-sink-to-tidb-cloud.md?plain=1#L1-L6